light_prop/propagation_input.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from light_prop.calculations import lens, gaussian
+
+
+class PropagationInput():
+    def __init__(self):
+        self.amplitude = None
+        self.phase = None
+
+    def from_image(self, amplitude_image, phase_image):
```

Comment:
No, I am strongly against it. Create util class which handles IO and produces valid data or reports errors (aka throws)

`lightField = light_prop.io.load('amp.png', 'phase.img')`

Reply:
do you see more io operations here? 
thinking if it should be `light_prop.io or light_prop.lightfield.io`

Reply:
`load`/`save` from/to:
- png
- bmp
- csv? 
- json?
- yml?
- custom format?

Maybe `upload` in the future

Reply:
yes

Reply:
No, just kidding :) Tbh bmp would be OK now